.github/pull_request_template.md

```diff
@@ -1,4 +1,4 @@
-**Are you a current Fivetran customer?** 
+**Are you a current Fivetran customer?**  
```

Comment:
It looks like we didn't include the maintainer PR templates. Can we add those as well.